### PR TITLE
Update create-external-load-balancer.md

### PR DESCRIPTION
**Pull request by JaccyLi**

### CODE REVIEW

**Comment by JaccyLi** [comment]
typo

File: content/zh/docs/tasks/access-application-cluster/create-external-load-balancer.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ that sends traffic to the correct port on your cluster nodes
 _provided your cluster runs in a supported environment and is configured with
 the correct cloud load balancer provider package_.
 -->
-创建服务时，您可以选择自动创建云网络负载均衡器。这提供了一个外部可访问的 IP 地址，可将流量分配到集群节点上的正确端口上 _假设集群在支持的环境中运行，并配置了正确的云负载平衡器提供商包_。
+创建服务时，您可以选择自动创建云网络负载均衡器。这提供了一个外部可访问的 IP 地址，可将流量分配到集群节点上的正确端口上 _假设集群在支持的环境中运行，并配置了正确的云负载平衡器提供商包。
```
Comment (tanjunchen) [inline content/zh/docs/tasks/access-application-cluster/create-external-load-balancer.md:36]: we need to update this ?
/assign @tengqm

Comment (tengqm) [reply content/zh/docs/tasks/access-application-cluster/create-external-load-balancer.md:36]: no. this change is invalid

Comment (SataQiu) [reply content/zh/docs/tasks/access-application-cluster/create-external-load-balancer.md:36]: @JaccyLi  `_XXX_` 这种形式，前后的短下划线是 Markdown 斜体表示语法，并非错误。

**Comment by tengqm** [comment]
/close

Closing because the fix is invalid.

**Pull request closed without merging**